docs/testplan/build_2tier_network.md

```diff
@@ -0,0 +1,386 @@
+
+T0-A Configuration:
+
+```json
+"FEATURE": {
+    "bgp": {
+        "auto_restart": "enabled",
+        "check_up_status": "false",
+        "delayed": "False",
+        "has_global_scope": "False",
+        "has_per_asic_scope": "True",
+        "high_mem_alert": "disabled",
+        "set_owner": "local",
+        "state": "enabled",
+        "support_syslog_rate_limit": "true"
+    }
+}
+```
+
+T1-A Configuration:
+
+```json
+"FEATURE": {
+    "bgp": {
+        "auto_restart": "enabled",
+        "check_up_status": "false",
+        "delayed": "False",
+        "has_global_scope": "False",
+        "has_per_asic_scope": "True",
+        "high_mem_alert": "disabled",
+        "set_owner": "local",
+        "state": "enabled",
+        "support_syslog_rate_limit": "true"
+    }
+}
+```
+
+#### Define Device Metadata
+
+Define the metadata of the two devices. Each switch must be correctly identified with its BGP Autonomous System Number (ASN) and router ID. T0 switches function as top-of-rack routers, while T1 switches serve as leaf routers.
+
+T0-A Configuration:
+
+```json
+"DEVICE_METADATA": {
+    "localhost": {
+        "bgp_asn": "64001",
+        "buffer_model": "traditional",
+        "cloudtype": "Public",
+        "default_bgp_status": "up",
+        "default_pfcwd_status": "enable",
+        "deployment_id": "1",
+        "docker_routing_config_mode": "separated",
+        "hostname": "switch-T0-A",
+        "hwsku": "ABCDEFG",
+        "region": "None",
+        "synchronous_mode": "enable",
+        "timezone": "UTC",
+        "type": "ToRRouter",
+        "yang_config_validation": "disable",
+        "bgp_router_id": "10.100.0.81"
+    }
+}
+```
+
+T1-A Configuration:
+
+```json
+"DEVICE_METADATA": {
+    "localhost": {
+        "bgp_asn": "65001",
+        "buffer_model": "traditional",
+        "cloudtype": "Public",
+        "default_bgp_status": "up",
+        "default_pfcwd_status": "enable",
+        "deployment_id": "1",
+        "docker_routing_config_mode": "separated",
+        "hostname": "switch-T1-A",
+        "hwsku": "ABCDEFG",
+        "region": "None",
+        "synchronous_mode": "enable",
+        "timezone": "UTC",
+        "type": "LeafRouter",
+        "yang_config_validation": "disable",
+        "bgp_router_id": "10.100.0.88"
+    }
+}
+```
+
+#### Configure Ports and Interfaces
+
+Each BGP session requires a pair of interfaces. Configure the two interfaces, ensuring they are in the same subnet.
+
+T0-A Configuration:
+
+```json
+"PORT": {
+    "Ethernet0": {
+        "admin_status": "up",
+        "alias": "Ethernet1/1",
+        "description": "Ethernet1/1",
+        "fec": "rs",
+        "index": "1",
+        "lanes": "17",
+        "mtu": "9100",
+        "pfc_asym": "off",
+        "speed": "100000",
+        "subport": "1",
+        "tpid": "0x8100"
+    }
+},
+"INTERFACE": {
+    "Ethernet0": {},
+    "Ethernet0|2001:db9::1:0:10/120": {}
```

Comment:
I believe the interface IP should be /126, which is what we usually do for p2p IP range.